akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -380,6 +380,10 @@ sealed trait SslTlsOutbound
  *  - `enabledProtocols` will be passed to `SSLEngine::setEnabledProtocols()`
  *  - `clientAuth` will be passed to `SSLEngine::setWantClientAuth()` or `SSLEngine.setNeedClientAuth()`, respectively
  *  - `sslParameters` will be passed to `SSLEngine::setSSLParameters()`
+ *
+ * Please note that passing `clientAuth = None` means that no change is done
```

Comment:
`HttpsContext` I'm intensely working on since monday as we discussed some in the last http call with @sirthias, effectively making it not require all the values for TLS config that it currently carries, as ssl-config is aimed to be doing that instead. I hope to push a PR today, and see also https://github.com/typesafehub/ssl-config for WIP on that side of things.

I see what you mean @rkuhn will have to double check I keep the wanted semantics.